usaspending_api/references/v2/views/agency.py

```diff
@@ -73,6 +73,8 @@ def get(self, request, pk, format=None):
             obligated_amount=Sum('obligations_incurred_total_by_tas_cpe'),
             outlay_amount=Sum('gross_outlay_amount_by_tas_cpe'))
 
+        cj = toptier_agency.justification if len(toptier_agency.justification) > 0 else None
```

Comment:
This fails if `justification` is None.  [Should probably do it like you did the other one](https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/references/v2/views/toptier_agencies.py#L105).

Reply:
Dang, knew I had overlooked something. Will fix.